### DISCUSSION TITLE
Hi, I'm new to Github! I already need some help!

### DISCUSSION THREAD

**Discussion by nhialmoses** [community/community#69707] [New to GitHub]
Hi, my name is Nhial and I'm new to GitHub and coding in general. I've been exploring to learn some code to help automate some repetitive task I've been assigned with. 

One of the things I'd like is a program that could go into a site like, https://verbalcommits.com/school-search/D1, and go and download at openly available information into an excell spreadsheet. A bonus would be a program that could click onto each school and download the current roster information. VerbalCommits displays the schools official site as well so id like to go in and download the coaches contact but I am unsure if that would require a whole different program. 

Would there be anybody willing to help point me into the right direction? what type of program I should learn to write these on? I was thinking python since I heard that would be easy to learn, but what you guys think? any help would be appreciated.

**Comment by TankEngine-ish** [comment]
Hello Nhial,

Python can be a good choice to begin your coding learning path but as I can tell by your questions you want to deal with data so a quick introduction to data structures and databases as well as SQL is worth looking into.
I suppose you're already familiar with IDE tools like VS Code?

**Reply by nhialmoses** [reply]
Thanks for getting back to me. Yes, I current have VS code on my Mac. luckily had a a brief chance to explore SQL in a work opportunity but im not too familiar with it. I'll definitely take a look. 

thanks!

**Comment by jjrh92** [comment]
Python sounds like the best to start, go on with the basics and there's plenty of resources online to learn it. Start by downloading vscode and then visit [python.org](https://www.python.org/) to get started.

**Reply by nhialmoses** [reply]
I will do that, Thanks!

**Comment by RebelAngel98** [comment]
Personally I started with JavaScript/HTML, it's one of the harder codes, but compared to Python it's like taking the SATs without prep. I'm the type of person to recommend the most advanced ones before going with the easier ones, just to 'give your brain a break' once in a while. I recommend using GeeksForGeeks or W3 Schools. Both websites are awesome! Also try CS Dojo on YouTube. A great way to get started as well, examine some codes and see what they do and how they do it. I found that has really helped me. 

Good luck on your programming journey! 

P.S. For your code, I think SQL would be a good choice if you need to show this to other people off of your laptop. SQL is very simple to learn and for the most part is very straight-forward. Python can do a lot, but if you need this in a way that I'm thinking, more like an application, I would do SQL. If you don't feel comfortable with SQL, C/C++ and Python are both excellent choices!
